src/vs/workbench/services/gettingStarted/common/gettingStartedContent.ts

```diff
@@ -42,141 +39,195 @@ type GettingStartedContent = GettingStartedCategory[];
+		content: {
+			type: 'items',
+			items: [
+				{
+					id: 'pickColorTheme',
+					title: localize('gettingStarted.pickColor.title', "Customize the Look With Themes"),
+					description: localize('gettingStarted.pickColor.description', "Adapt VS Code to your taste with themes, customizing interface and language syntax colors."),
+					button: { title: localize('gettingStarted.pickColor.button', "Browse Color Themes"), command: 'workbench.action.selectTheme' },
+					doneOn: { eventFired: 'themeSelected' },
+					media: { type: 'image', altText: 'Color theme preview for dark and light theme.', path: 'colorTheme.jpg', }
+				},
 				{
 					id: 'findLanguageExtensions',
-					description: localize('findLanguageExtsTask.description', "Get support for your languages like JavaScript, Python, Java, Azure, Docker, and more."),
-					title: localize('findLanguageExtsTask.title', "Languages & Tools"),
+					title: localize('gettingStarted.findLanguageExts.title', "Add More Language & Tools Support"),
+					description: localize('gettingStarted.findLanguageExts.description', "Install extensions with one click to support additional languages like Python, Java, Azure, Docker, and more."),
 					button: {
-						title: localize('findLanguageExtsTask.button', "Install Language Support"),
+						title: localize('gettingStarted.findLanguageExts.button', "Browse Language Extensions"),
 						command: 'workbench.extensions.action.showLanguageExtensions',
 					},
 					doneOn: { commandExecuted: 'workbench.extensions.action.showLanguageExtensions' },
-					media: { type: 'image', altText: 'Languages', path: 'Languages.jpg', }
+					media: { type: 'image', altText: 'Language extensions', path: 'languageExtensions.jpg', }
+				},
+				{
+					id: 'githubLogin',
+					title: localize('gettingStarted.githubLogin.title', "Use GitHub in VS Code"),
+					description: localize('gettingStarted.githubLogin.description', "Integrated GitHub makes it easier for you to manage projects from inside your code editor, including authentication, publishing repos, and viewing your repo timeline."),
+					when: '!githubBrowser:hasProviders',
+					button: {
+						title: localize('gettingStarted.githubLogin.button', "Sign in to GitHub"),
+						command: 'githubsignIn',
+					},
+					doneOn: { commandExecuted: 'workbench.extensions.action.showLanguageExtensions' },
+					media: { type: 'image', altText: 'Commiting a change via Git in VS Code.', path: 'github.jpg', }
 				},
-
 				{
 					id: 'pickAFolderTask-Mac',
-					description: localize('gettingStartedOpenFolder.description', "Open a project folder to get started!"),
-					title: localize('gettingStartedOpenFolder.title', "Open Folder"),
-					when: 'isMac',
+					title: localize('gettingStarted.setup.OpenFolder.title', "Open Your Project"),
+					description: localize('gettingStarted.setup.OpenFolder.description', "Open a project folder to get started!"),
+					when: '!emptyWorkspaceSupport && isMac',
```

Comment:
Why `when: !emptyWorkspaceSupport` here? Given the category is `when: emptyWorkspaceSupport`, this will never show yes? (Also, why is the category `when: emptyWorksapceSupport`?)